File: HelloUser.java
import java.util.Scanner;
public class Hellousername{
    public static void main(String[] args){
        String name;
        Scanner input = new Scanner(System.in);
        System.out.print("Enter username: ");
        name = input.next();
        System.out.println("Hello "+name);
    }
}

File: HelloUser.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{
    string name;                        //char name[20];
    cout<<"Enter your name : ";
    getline(cin,name);                  // getline() is a inbuilt function in c++
    //cout<<"Name is "+name+"\n";
    //cout<<"Name is "<<name<<"\n";
    cout<<"Hello "<<name<<endl;
    //return 0;
}